File: openapi/code_samples/go/api_v1_passwordless_email_verify/post.go
// Verify with OTP code
verifyResponse, err := scalekitClient.Passwordless().VerifyPasswordlessEmail(
    ctx,
    &scalekit.VerifyPasswordlessOptions{
        Code:          "123456", // OTP code
        AuthRequestId: authRequestId,
    },
)

if err != nil {
    // Handle error
    return
}

// Verify with magic link token
verifyResponse, err := scalekitClient.Passwordless().VerifyPasswordlessEmail(
    ctx,
    &scalekit.VerifyPasswordlessOptions{
        LinkToken: linkToken, // Magic link token
    },
)

if err != nil {
    // Handle error
    return
}

// User verified successfully
userEmail := verifyResponse.Email

File: openapi/code_samples/javascript/api_v1_passwordless_email_verify/post.js
const { authRequestId } = sendResponse

// Verify with OTP code
const verifyResponse = await scalekit.passwordless.verifyPasswordlessEmail(
  { code: '123456' },
  authRequestId,
)

// Or verify with magic link token
const linkVerifyResponse = await scalekit.passwordless.verifyPasswordlessEmail({
  linkToken: linkToken,
})
